site/src/components/WorkspaceScheduleForm/WorkspaceScheduleForm.tsx

```diff
@@ -41,6 +41,7 @@ export const Language = {
   errorTime: "Time must be in HH:mm format (24 hours)",
   errorTimezone: "Invalid timezone",
   errorNoStop: "Time until shutdown must be greater than zero when auto-stop is enabled",
+  errorTtlMax: "Please enter a limit that is less than or equal to 168 hours (7 days).",
```

Comment:
nit: this line has punctuation but none of the others in the diff view do. doesn't matter which we use as long as we stay consistent IMO 
```suggestion
  errorTtlMax: "Please enter a limit that is less than or equal to 168 hours (7 days)",
```